src/cheat_sheet.py

```diff
@@ -0,0 +1,322 @@
+from talon import Module, ui, registry, skia, actions
+from talon.canvas import Canvas
+import re
+import webbrowser
+
+mod = Module()
+mod.mode("cursorless_cheat_sheet", "Mode for showing cursorless cheat sheet gui")
+cheat_sheet = None
+last_mouse_pos = None
+
+line_height = 34
+text_size = 16
+close_size = 24
+padding = 4
+instructions_url = "https://github.com/pokey/cursorless-talon/tree/master/docs"
+
+
+class CheatSheet:
+    def __init__(self):
+        screen = ui.main_screen()
+        width = 1410
```

Comment:
Assuming this width is fine on all screens?

Reply:
Well no. It can absolutely be too wide for some screens. The only solution for that would be to adapt the font size depending on the user screen resolution